plugins/modules/mail.py

```diff
@@ -214,6 +214,15 @@
     subject: Ansible-report
     body: System {{ ansible_hostname }} has been successfully provisioned.
     secure: starttls
+
+ - name: Sending an email with a timeout
+   community.general.mail:
+    host: localhost
+    port: 25
+    to: John Smith <john.smith@example.com>
+    subject: Ansible-report
+    body: System {{ ansible_hostname }} has been successfully provisioned.
+    timeout: 10  
```

Comment:
Same here, I don't see any value in simply repeating the previous example with a minimal modification. How about adding this modification to the previous example and change its name to "... custom EHLO, reduced timeout"?

Reply:
Updated as per your recommendations